locust/stats.py

```diff
@@ -888,8 +890,10 @@ def stats_history(runner: "Runner") -> None:
                 "time": datetime.datetime.now(tz=datetime.timezone.utc).strftime("%H:%M:%S"),
                 "current_rps": stats.total.current_rps or 0,
                 "current_fail_per_sec": stats.total.current_fail_per_sec or 0,
-                "response_time_percentile_95": stats.total.get_current_response_time_percentile(0.95) or 0,
-                "response_time_percentile_50": stats.total.get_current_response_time_percentile(0.5) or 0,
+                "response_time_percentile_95": stats.total.get_current_response_time_percentile(PERCENTILES_TO_CHART[0])
```

Comment:
Variables needs to be changed. Probably use a list: response_time_percentile[0] instead of response_time_percentile_95.

Reply:
(If that cannot be done, it could be response_time_percentile_first/second or something)

Reply:
Sure, will use the variable

Reply:
👍